LocalSettings.php

```diff
@@ -3270,6 +3270,11 @@
 		'+elementswiki' => array(
 			'founder' => array(
 				'read' => true,
+				'globalblock-whitelist' => true,
+				'interwiki' => true,
```

Comment:
Interwiki is not given to userrights on wikis as it edits Miraheze's global interwiki.